Cargo.toml

```diff
@@ -33,13 +33,7 @@ rust-embed = { version = "6.6.0", optional = true, features = [
     "interpolate-folder-path",
     "include-exclude",
 ] }
-wasmer = { version = "2.3", optional = true, default-features = false, features = [
-    "sys-default",
-    "cranelift",
-    "default-compiler",
-    "default-cranelift",
-    "default-universal",
-] }
+wasmer = { version = "3.3", optional = true }
```

Comment:
Doesn't wasmer need features or are the defaults fine?

Reply:
The defaults seem to be just fine tbh. I can compile nargo against this branch and then interact with Noir programs.